auth0/src/main/java/com/auth0/android/provider/CustomTabsController.java

```diff
@@ -126,7 +126,6 @@ public void run() {
                 final Intent intent = new CustomTabsIntent.Builder(session.get())
                         .build()
                         .intent;
-                intent.addFlags(Intent.FLAG_ACTIVITY_NO_HISTORY);
```

Comment:
Removing does not cause any other issues? Only asking as issues that required this to fix in RNA.

Reply:
CCT is not another tab that stays open on the browser, is more like an activity. By removing the flag you make the activity stay up after the app is paused/resumed. Yes, we should probably do the same for RNA and just keep that flag for browser (not CCT) authentication. I didn't do the PR yet because it's not just removing the flag.. CustomTabsIntent may fail to find a package that could "handle CCT and so default to use Browser. We must catch and add the flag to that case too.